bin/make_dht_configs

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import argparse
+import json
+import os
+
+from syncr_backend.constants import DEFAULT_DPS_CONFIG_FILE
+from syncr_backend.constants import DEFAULT_PKS_CONFIG_FILE
+from syncr_backend.init.node_init import get_full_init_directory
+
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "listenport",
+        type=str,
+        help='Port to listen on for DHT',
+    )
+    parser.add_argument(
```

Comment:
how about having `--bootstrap-peers` in the format `./make_dht_configs 99999 --bootstrap-peers 1.2.3.4:5678,9.0.1.2:3456`?

Reply:
It might be confusing having separate ip and port lists